scripts/codegen-check-consistency.js

```diff
@@ -0,0 +1,4 @@
+const { checkCodegenIntegrity } = require('./codegen-utils');
+
+ checkCodegenIntegrity();
```

Comment:
no newline

Reply:
fixed in: https://github.com/software-mansion/react-native-svg/pull/2357/commits/b20a8c8157363c41523bf15b1ff480bc0ad133e3